crowdloan/polkadot.json

```diff
@@ -296,5 +296,14 @@
     "description": "Hashed Network enables the full lifecycle for digitally-native organizations and web3-curious businesses seeking benefits of decentralized digital economies",
     "website": "https://hashed.network/",
     "icon": "https://raw.githubusercontent.com/nova-wallet/nova-utils/master/icons/chains/gradient/Hashed.svg"
+  },
+  {
+    "paraid": "2094",
+    "name": "Pendulum",
+    "token": "PEN",
+    "description": "Pendulum is an open-source blockchain that aims to establish the missing link between the fiat and the DeFi ecosystems through a fiat-optimized smart contract blockchain.Pendulum is built with Substrate and will allow connecting DeFi applications to the foreign exchange market, building automated market makers (AMMs) to introduce scalable liquidity pools for fiat currencies, and create yield earning opportunities for fiat token holders.",
+    "website": "https://pendulumchain.org/",
+    "icon": "https://raw.githubusercontent.com/nova-wallet/nova-utils/master/icons/chains/gradient/Pendulum.svg",
+    "rewardRate": 11
```

Comment:
Are you sure about rate?
it means that for 1 DOT you will receive 11 PEN

Reply:
Yes, Please check here https://parachains.info/auctions/polkadot-33-39